Admin/AbstractAdmin.php

```diff
@@ -2816,6 +2659,163 @@ public function getDashboardActions()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    protected function configure()
```

Comment:
I don't understand what you mean by that.

Yes, this method was in the `AdminInterface`, because it was a public method. When making this method `abstract` every `Admin` class must implement this. IMO that's not the idea behind this method.